setup.py

```diff
@@ -1,11 +1,8 @@
 from setuptools import setup
 
 setup(
-      name="mbuild_ONA",
-      install_requires="mbuild",
-      entry_points={
-                    "mbuild.plugins":[ "ONA_box = mbuild_ONA.mbuild_ONA:ONA_box"
-                        ]
-                    },
-                    py_modules=["mbuild_ONA"],
-                        )
+    name="mbuild_ona",
+    install_requires="mbuild",
+    entry_points={"mbuild.plugins": ["DNA = mbuild_ona.mbuild_ona:DNA"]},
```

Comment:
Should there also be an entry point for the `ONA_box` as well?  It would be like this:

```
entry_points={"mbuild.plugins": [
"DNA = mbuild_ona.mbuild_ona:DNA"],
"ONA = mbuild_ona.mbuild_ona:ONA_box",
]
}
```